browser/src/map/handler/Map.TouchGesture.js

```diff
@@ -98,44 +98,60 @@ L.Map.TouchGesture = L.Handler.extend({
 		}
 	},
 
+	_boundEventListeners: {},
+
+	bindEventListener: function(f, context, touchOnly) {
+		if (this._boundEventListeners[L.stamp(f) + '-' + L.stamp(context) + '-' + L.stamp(touchOnly)] === undefined) {
+			var bound = L.bind(f, context);
+
+			if (touchOnly) {
+				bound = window.touch.touchOnly(bound);
+			}
+
+			this._boundEventListeners[L.stamp(f) + '-' + L.stamp(context) + '-' + L.stamp(touchOnly)] = bound;
+		}
+
+		return this._boundEventListeners[L.stamp(f) + '-' + L.stamp(context) + '-' + L.stamp(touchOnly)];
```

Comment:
this is also the mentioned repeated piece?

Reply:
yes, it's quite similar ... I'll make something to generalize all these 3, because it's quite a useful thing to have